app/assets/stylesheets/theme_previews.css

```diff
@@ -19,7 +19,7 @@ input[type='radio']:checked + label .theme-content {
   cursor: pointer;
   border-radius: 10px;
   height: 800px;
-  width: 600px;
+  max-width: 600px;
```

Comment:
addressed ur comment - this change will make it so that no theme will exceep 600px (it's standard size for pdf) on larger devices. 

To test you'll want to check all 3 pages where the resume is rendered.